tools/runners/test262-harness.py

```diff
@@ -344,6 +391,8 @@ def build_options():
                       help="The command-line to run")
     result.add_option("--tests", default=path.abspath('.'),
                       help="Path to the tests")
+    result.add_option('--es51', default=False, action='store_true',
```

Comment:
How about `--es5.1` ? Personally I think the `51` is quite deceptive.

Reply:
how to do with self.suite.es51?
self.suite.es5_1?

Reply:
I'm fine with `es5_1` as well.

Reply:
> I'm fine with `es5_1` as well.

I have an extra problem, the es51 option are compiled from `run-test-suite-test262.py`
```
    group.add_argument('--es51', default=False, const='default',
                       nargs='?', choices=['default', 'all', 'update'],
                       help='Run test262 - ES5.1. default: all tests except excludelist, ' +
                       'all: all tests, update: all tests and update excludelist')
```

That's why I use it, maybe we do the rename in separate patch?

Reply:
> I'm fine with `es5_1` as well.

renamed to es5_1:)